### ISSUE TITLE
Some wireless interfaces can only be configured while up, others only while down.

### ISSUE THREAD

**Issue by virtadpt** [Byzantium/Byzantium#235]
Some wireless interfaces can only be configured while up (`ifconfig up`), others only while down (`ifconfig down`).  We often see strings of errors of this sort:

```
Attempting to configure interface wlan0.
Attempting to configure the wireless interface. Try: 0
Error for wireless request "Set Mode" (8B06) :
SET failed on device wlan0 ; Operation not permitted.
Error for wireless request "Set ESSID" (8B1A) :
SET failed on device wlan0 ; Operation not permitted.
Error for wireless request "Set AP Address" (8B14) :
SET failed on device wlan0 ; Operation not permitted.
Failed to setup the interface properly. Retrying...
```

Since byzantium_configd.py now tries only three times to configure a network interface and dies if it can't, I propose the following:

Try to configure the network interface with the interface turned down.  If it is unable to do so, run `ifconfig interface up` and then try to configure it again.  If, after this second set of three attempts it is unable to do so, then ABEND.

**Comment by zensur** [comment]
Why do you suppose that is?

**Comment by virtadpt** [comment]
Usually, the firmware the driver loads.  Some do it one way, some do it another, and I guess others let whatever the compiler that generates the firmware blob itself decide.  It's very inconsistent across all of my test hardware.

**Comment by virtadpt** [comment]
Fixed in commit 0f3fa77bae71e6810fce96d5d46411d0aee61be0.  Changes tested on ByzPi, ported over to Byzantium.
